HyperV/scripts/create-environment.ps1

```diff
@@ -157,8 +164,8 @@ if (Test-Path $pythonDir)
 }
 Write-Host "Ensure Python folder is up to date"
 Write-Host "Extracting archive.."
-& "C:\Program Files\7-Zip\7z.exe" x -y "$pythonArchive"
-& "C:\Program Files\7-Zip\7z.exe" x -y python27new.tar
+& $7zExec x -y "$pythonArchive"
+& $7zExec x -y "$pythonTar"

```

Comment:
Yep, we do: 
if (Test-Path $pythonArchive)
{ Remove-Item -Force $pythonArchive}
and for tar also :)